### PR TITLE
Ec/feature/render header

### PR DESCRIPTION
**Pull request by mainlyetcetera**
#### Header
- adds `h1` title of app to `Header` to render
- downloads `react-router-dom`
  - this is initially for the `Link`
- adds `Link` to `Faq`
  -  this is anticipating a specific route called `Faq`
- adds the `SearchBar` component
- renders the `SearchBar` component in the `Header`

#### SearchBar
- uses hook to give `SearchBar` state and update accordingly
- has state update upon typing in the search bar
- adds submit button
- upon clicking submit button, 
  - anticipates potentially sending state of search to `App` to use api call to render proper data
  - this is up for discussion
- clears search bar upon submission
### Is this a feature or Fix/Refactor?  
- feature
- We need to decide whether `SearchBar` or `App` is responsible for handling API requests
- `src/Components/Header/Header.js`
- `src/Components/SearchBar/SearchBar.js`
### How should this be tested?
- once `App` is created
  - pull down this branch (not to main)
  - observe `App` rendering `Header`
  - observe `Header` rendering the title, a link, and the `SearchBar`
>What steps need to be taken to confirm functionality? 
- none yet, as functionality is dependent on other components' existence
- once `App` is made, styling can occur, as then `Header` will actually render
- This affects #5, but should not close it yet as the `Header` is not complete without the `SearchBar/App` API logic decided

### CODE REVIEW

File: src/Components/SearchBar/SearchBar.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+      <button
+        onClick={event => submitSearch(event)}
+      >
+        Search!
+      </button>
+    </form>
+  )
+}
+
+export default SearchBar;
```
Comment (deadbelly) [inline src/Components/SearchBar/SearchBar.js:36]: I love this search bar. I think we may need it to change depending on how we run the search but it's a great boilerplate.

File: src/Components/Header/Header.js [hunk 2]
```diff
@@ -1,8 +1,15 @@
...
 
-const Header = () => {
+import SearchBar from '../SearchBar/SearchBar.js';
 
-}
+const Header = () => (
+  <div>
+    <h1>BreathEZ</h1>
+    <Link to='/faq'>FAQ</Link>
+    <SearchBar />
```
Comment (deadbelly) [inline src/Components/Header/Header.js:11]: It's possible that I'll want to re-organize this as the functionality progresses but I think we shouldn't worry about it now. For at least the next two days this is great.

**Review by deadbelly — APPROVED** [review]
This looks great! I appreciate you importing Router as well!

**Review by thaomonster — APPROVED** [review]

**Comment by deadbelly** [comment]
The merge conflict here seems to be pretty difficult to deal with. Let's close this PR and open a new on a fresh branch to avoid this issue.

**Pull request closed without merging**